Automattic/jetpack
# Updates Blaze entrypoint to be compatible with Woo Blaze

With the work being done to add a customized version of Blaze for Woo merchants, we want to make the Blaze module compatible with the new plugin. 

The main change here is that Woo Blaze will take over the menu and entry points if the new Woo Blaze plugin is installed on the site. The main idea here is to present the new merchant-customized version of Blaze if we detect WooCommerce and Woo Blaze active.

## Proposed changes:

* Hide the dashboard menu option if WooCommerce and WooBlaze are active. The new plugin will handle the menu and live under the Marketing section with other Ads marketing channels like Google and Facebook.
* Change the redirection URL to the management campaign widget, depending if WooBlaze is active

### Other information:


## Jetpack product discussion


peeHDf-2hF-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






You will need to test this on a site with WooCommerce and the new Woo Blaze plugin. Please ping me to get the Woo Blaze plugin's zip file (it has yet to be released).

The testing steps are the same for Atomic and self-hosting.
* Verify that the Tools->Advertising menu option is not shown
* Go to Posts->All Posts and verify that the "**Promote with Blaze**" is shown in the posts actions
* Click the "**Promote with Blaze**" button and verify that you land on the new Woo Blaze customized dashboard (it is inside the marketing section).
* Now go To Posts->All Posts and create a new post
* Publish that post, and verify that the "**Promote with Blaze**" publish widget is shown.
* Click on "Blaze this post" and verify that you land on the new Woo Blaze customized dashboard.

Now, run a smoke test on a regular site (without the Woo Blaze plugin) and verify that the Blaze dashboard works correctly (the menu option is shown and links from Posts->All Posts redirects to the correct dashboard).
